files/motd

```diff
@@ -1,6 +1,6 @@
-'|.   '|'   ..|'''.|     |     |''||''|  .|'''.|  
- |'|   |  .|'     '     |||       ||     ||..  '  
- | '|. |  ||           |  ||      ||      ''|||.  
- |   |||  '|.      .  .''''|.     ||    .     '|| 
-.|.   '|   ''|....'  .|.  .||.   .||.   |'....|'  
-
+  [0;1;31;91m..[0;1;33;93m|'[0;1;32;92m''[0;1;36;96m.|[0m [0;1;34;94m'[0;1;35;95m||[0;1;31;91m'[0m  [0;1;33;93m.[0;1;32;92m|'[0;1;36;96m''[0;1;34;94m.|[0m      [0;1;32;92m|[0m
```

Comment:
Since you added that sweet plain text, we could just check for that right?  That should be good enough, no?